server/util/twilio.js

```diff
@@ -0,0 +1,9 @@
+import Twilio from 'twilio';
+
+// Account SID and Auth Token are stored as environmental variables
+// Twilio Node module will check for them automatically upon initialization
+const twilioClient = new Twilio();
+const CALLER_ID = process.env.TWILIO_CALLER_ID;
+
+console.log(twilioClient);
```

Comment:
Maybe add a comment here so the first person to add a method should remove these?

Reply:
Will do